### DISCUSSION TITLE
DAX studio connection via msmdpump.dll

### DISCUSSION THREAD

**Discussion by danzie10a** [DaxStudio/DaxStudio#608] [General]
Is it possible to connect to SSAS Tabular instance via msmdpump.dll http end point?
It works when I attempt to connect from SQL Managing Studio to that end point, but DAX Studio is not able to connect.
Is this a limitation? Any workarounds?

Thanks

**Comment by dgosbell** [comment]
Yes, it is possible and it should work. We have one of these endpoints setup at my workplace and I just double checked and it worked fine.

What sort of authentication are you using? If it's windows auth it should just work, if you are using basic auth you would need to expand the Advanced options and add `User ID=<user>;Password=<password>` to the Additional Options section
![image](https://user-images.githubusercontent.com/8241830/117219759-46498100-ae49-11eb-8018-800b57f472d7.png)

When you say it's not able to connect - what sort of error / warning are you seeing?

**Comment by danzie10a** [comment]
Yes, that did it. I did not know I could pass credentials in "Additional Options".
I was expecting DAX studio to give me credentials pop up window.

Thank you.
